File: AKNOVABROW/Models/VPNServer.cs
namespace AKNOVABROW.Models
{
    public class VPNServer
    {
        public string Country { get; set; } = "";
        public string Flag { get; set; } = "";
        public string ProxyHost { get; set; } = "";
        public int ProxyPort { get; set; }
        public string Speed { get; set; } = "";

        public override string ToString() => $"{Flag} {Country} ({Speed})";
    }
}

File: aknovabrow_mobile/lib/models/vpn_server.dart
class VPNServer {
  final String country;
  final String flag;
  final String speed;

  VPNServer({
    required this.country,
    required this.flag,
    required this.speed,
  });

  String get displayName => '$flag $country ($speed)';

  static List<VPNServer> getServers() => [
        VPNServer(country: 'United States', flag: '🇺🇸', speed: 'Fast'),
        VPNServer(country: 'United Kingdom', flag: '🇬🇧', speed: 'Fast'),
        VPNServer(country: 'Germany', flag: '🇩🇪', speed: 'Medium'),
        VPNServer(country: 'Japan', flag: '🇯🇵', speed: 'Fast'),
        VPNServer(country: 'Singapore', flag: '🇸🇬', speed: 'Fast'),
        VPNServer(country: 'France', flag: '🇫🇷', speed: 'Medium'),
        VPNServer(country: 'Netherlands', flag: '🇳🇱', speed: 'Fast'),
        VPNServer(country: 'Australia', flag: '🇦🇺', speed: 'Medium'),
      ];
}